@codex please fix

```
 ✓ test/web-plugins.test.js (5 tests) 400ms
 ✓ test/web-schemas.test.js (40 tests) 8ms
 ✓ test/web-config.test.js (10 tests) 17ms
 ✓ test/lifecycle.test.js (13 tests) 33ms
 ✓ test/deliverables.test.js (7 tests) 40ms
 ✓ test/notifications.test.js (5 tests) 49ms
 ✓ test/web-status-hub-frontend.test.js (3 tests) 618ms
   ✓ status hub frontend > navigates to Applications and renders shortlist rows  344ms
 ✓ test/listings.test.js (5 tests) 9ms
 ✓ test/intake.test.js (9 tests) 19ms
 ✓ test/resume.test.js (14 tests) 20ms
 ✓ test/application-events.test.js (10 tests) 15ms
 ✓ test/web-session-security.test.js (5 tests) 78ms
 ✓ test/greenhouse.test.js (6 tests) 85ms
 ✓ test/intake-plan.test.js (5 tests) 18ms
 ✓ test/workable.test.js (4 tests) 2511ms
   ✓ Workable ingest > fetches Workable jobs and writes snapshots  514ms
   ✓ Workable ingest > sends authorization header when JOBBOT_WORKABLE_TOKEN is set  999ms
   ✓ Workable ingest > throws when a job detail fetch fails  998ms
 ✓ test/activity-insights.test.js (7 tests) 7ms
 ✓ test/parser.test.js (19 tests) 3ms
 ✓ test/client-payload-store.test.js (9 tests) 3ms
 ✓ test/listings.e2e.test.js (1 test) 14ms
 ✓ test/job-source-adapters.test.js (5 tests) 19ms
 ✓ test/experiments.test.js (6 tests) 6ms
 ✓ test/cli-audit-exports.test.js (3 tests) 181ms
 ✓ test/opportunities-backup-restore.test.js (1 test) 205ms
 ✓ test/resume-pipeline.test.js (5 tests) 6ms
 ✓ test/profile-import.test.js (3 tests) 14ms
 ✓ test/cli-listings.test.js (3 tests) 3ms
 ✓ test/web-audits.test.js (5 tests) 855ms
   ✓ web interface audits > meets the documented accessibility and performance baselines  772ms
 ✓ test/discards.test.js (6 tests) 8ms
 ✓ test/web-server-realtime.test.js (2 tests) 47ms
 ✓ test/speech.test.js (5 tests) 13ms
 ✓ test/parser.fields.perf.test.js (1 test) 1077ms
   ✓ parseJobText field scanning performance > outperforms the legacy field scanner  1076ms
 ✓ test/web-server-contracts.test.js (1 test) 2042ms
   ✓ web server CLI contracts > matches analytics funnel CLI output when filters are provided  2042ms
 ✓ test/web-server-integration.test.js (1 test) 289ms
 ✓ test/jobs.test.js (4 tests) 8ms
 ✓ test/smartrecruiters.test.js (2 tests) 1005ms
   ✓ SmartRecruiters ingest > fetches SmartRecruiters jobs and writes snapshots  510ms
   ✓ SmartRecruiters ingest > throws when the postings fetch fails  495ms
 ✓ test/resolve-allow-remote-access.test.js (15 tests) 3ms
 ✓ test/security-risk-assessment.test.js (3 tests) 2ms
 ✓ test/index.test.js (23 tests) 2ms
 ✓ test/audit-log-integrity.test.js (6 tests) 12ms
 ✓ test/web-security-regressions.test.js (2 tests) 81ms
 ✓ test/chore-prompts.test.js (4 tests) 5407ms
   ✓ chore:prompts > runs the prompts chore script successfully  1642ms
   ✓ chore:prompts > fails when prompt docs contain broken local links  1753ms
   ✓ chore:prompts > passes when a prompt doc deletion is staged  1746ms
 ✓ test/http-resilience.test.js (4 tests) 4ms
 ✓ test/services-http.test.js (3 tests) 10ms
 ✓ test/prompt-injection.test.js (3 tests) 11ms
 ✓ test/cli-max-bytes.test.js (3 tests) 37ms
 ✓ test/provider-tokens.test.js (3 tests) 11ms
 ✓ test/ashby.test.js (2 tests) 8ms
 ❯ test/web-server-remote-access.test.js (6 tests | 2 failed) 42ms
   ✓ web server remote access guard > rejects non-loopback hosts unless remote access is enabled 0ms
   ✓ web server remote access guard > treats string "false" flags as disabled to avoid accidental exposure 0ms
   × web server remote access guard > permits non-loopback hosts when explicitly enabled 40ms
     → Remote web command access requires authentication; configure JOBBOT_WEB_AUTH_TOKENS or JOBBOT_WEB_AUTH_TOKEN, or pass auth tokens when allowRemoteAccess is enabled.
   ✓ web server remote access guard > rejects malformed or empty flags even when the environment opts in 0ms
   × web server remote access guard > permits non-loopback hosts when the environment explicitly opts in 1ms
     → Remote web command access requires authentication; configure JOBBOT_WEB_AUTH_TOKENS or JOBBOT_WEB_AUTH_TOKEN, or pass auth tokens when allowRemoteAccess is enabled.
   ✓ web server remote access guard > rejects non-loopback hosts when the environment disables remote bindings 0ms
 ✓ test/http-client-manifest.test.js (2 tests) 5ms
 ✓ test/scripts-summarize-jobs.test.js (3 tests) 197ms
 ✓ test/jobs-adapters-common.test.js (4 tests) 3ms
 ✓ test/polish-refactor-plan-doc.test.js (3 tests) 2ms
 ✓ test/web-e2e.test.js (1 test) 13ms
 ✓ test/inference-config.test.js (5 tests) 4ms
 ✓ test/env-loading.test.js (2 tests) 3ms
 ✓ test/docs-adr.test.js (2 tests) 2ms
 ✓ test/golden-matches.test.js (7 tests) 7ms
 ✓ test/console-font.test.js (6 tests) 6ms
 ✓ test/scheduler.test.js (4 tests) 3ms
 ✓ test/web-documentation-storybook.test.js (5 tests) 5ms
 ✓ test/security-risk-assessment-cli.test.js (3 tests) 104ms
 ✓ test/web-deployment.test.js (4 tests) 2ms
 ✓ test/soc2-controls.test.js (3 tests) 5ms
 ✓ test/reminders-calendar.test.js (2 tests) 1ms
 ✓ test/match.test.js (4 tests) 2ms
 ✓ test/cover-letter.test.js (2 tests) 1ms
 ✓ test/lever.test.js (2 tests) 8ms
 ✓ test/feedback.test.js (4 tests) 5ms
 ✓ test/ingest.recruiterEmail.test.ts (1 test) 43ms
 ✓ test/resume-pdf-artifact.test.js (3 tests) 5ms
 ✓ test/summarize.baseline.test.js (12 tests) 1ms
 ✓ test/i18n.test.js (8 tests) 14ms
 ❯ test/web-log-transport.test.js (2 tests | 1 failed) 12ms
   × web server log transport > rejects insecure log transport URLs when binding to non-loopback hosts 3ms
     → expected [Function] to throw error matching /https/i but got 'Remote web command access requires au…'
   ✓ web server log transport > sends redacted telemetry entries to the configured transport 9ms
 ✓ test/docs-date-accuracy.test.js (1 test) 8ms
 ✓ test/settings.test.js (4 tests) 3ms
 ✓ test/web-scheduler-status.test.js (1 test) 9ms
 ✓ test/web-rate-limit-fuzzer.test.js (1 test) 5ms
 ✓ test/web-security-roadmap-doc.test.js (3 tests) 2ms
 ✓ test/web-security-alerts.test.js (1 test) 48ms
 ✓ test/web-health-checks.test.js (5 tests) 293ms
 ✓ test/chore-catalog.test.js (3 tests) 1ms
 ❯ test/web-server-host-guard.test.js (4 tests | 2 failed) 2ms
   ✓ web server host guard > rejects non-loopback hosts without an explicit override 0ms
   × web server host guard > allows remote hosts when explicitly enabled 1ms
     → Remote web command access requires authentication; configure JOBBOT_WEB_AUTH_TOKENS or JOBBOT_WEB_AUTH_TOKEN, or pass auth tokens when allowRemoteAccess is enabled.
   × web server host guard > accepts truthy environment values for remote access 0ms
     → Remote web command access requires authentication; configure JOBBOT_WEB_AUTH_TOKENS or JOBBOT_WEB_AUTH_TOKEN, or pass auth tokens when allowRemoteAccess is enabled.
   ✓ web server host guard > treats falsy environment values as an explicit deny 0ms
 ✓ test/resolve-native-cli-flag.test.js (6 tests) 1ms
 ✓ test/web-native-cli-disabled.test.js (1 test) 36ms
 ✓ test/web-session-manager.test.js (1 test) 1ms
 ✓ test/profile.test.js (2 tests) 5ms
 ✓ test/flash_and_report.test.js (1 test) 31ms
 ✓ test/sentence-extractor.test.js (3 tests) 1ms
 ✓ test/resume-preview.test.js (2 tests) 1ms
 ✓ test/docs-backup-restore.test.js (3 tests) 1ms
 ✓ test/web-ux-guidelines-doc.test.js (2 tests) 1ms
 ✓ test/web-release-prep-doc.test.js (1 test) 1ms
 ✓ test/pre-commit-hooks.test.js (1 test) 1ms
 ✓ test/docs-getting-started.test.js (3 tests) 1ms
 ✓ test/lifecycle.sankey.test.ts (1 test) 1ms
 ✓ test/docs-security-risk-assessment.test.js (2 tests) 0ms
 ✓ test/architecture-doc.test.js (2 tests) 1ms
 ✓ test/scoring.resume.perf.test.js (1 test) 35ms
 ✓ test/security-doc-threat-model.test.js (1 test) 0ms
 ✓ test/readme-onboarding.test.js (1 test) 1ms
 ✓ test/web-non-functional-doc.test.js (1 test) 1ms
 ✓ test/chore-reminders.test.js (1 test) 37ms
 ✓ test/docs-deployment-guide.test.js (2 tests) 1ms
 ✓ test/docs-configuration-cookbook.test.js (2 tests) 0ms
 ✓ test/user-journeys-doc.test.js (1 test) 1ms
 ✓ test/resume-pipeline-doc.test.js (1 test) 0ms
 ✓ test/web-client-payload-store.test.js (2 tests) 1ms
 ✓ test/scoring.requirements.perf.test.js (1 test) 128ms
 ✓ test/job-source-adapter-doc.test.js (1 test) 0ms
 ✓ test/web-command-registry.test.js (2 tests) 0ms
 ✓ test/module-event-bus.test.js (2 tests) 1ms
 ✓ test/install-console-font.test.js (1 test) 35ms
 ✓ test/scoring.unique.perf.test.js (1 test) 114ms
 ✓ test/parser.requirements.perf.test.js (1 test) 282ms
 ✓ test/scoring.perf.test.js (1 test) 120ms
 ✓ test/scoring.large.perf.test.js (1 test) 104ms
 ✓ test/summarize.perf.test.js (1 test) 115ms
 ✓ test/readme-http-docs.test.js (1 test) 0ms
 ✓ test/fetch-docs-link.test.js (1 test) 0ms
 ✓ test/package-export.test.js (1 test) 0ms
 ✓ test/summarize.repeat.perf.test.js (1 test) 106ms
 ✓ test/summarize.whitespace.perf.test.js (1 test) 8ms
 ✓ test/vitest-config.test.js (1 test) 73ms

⎯⎯⎯⎯⎯⎯⎯ Failed Tests 5 ⎯⎯⎯⎯⎯⎯⎯

 FAIL  test/web-log-transport.test.js > web server log transport > rejects insecure log transport URLs when binding to non-loopback hosts
AssertionError: expected [Function] to throw error matching /https/i but got 'Remote web command access requires au…'

- Expected: 
/https/i

+ Received: 
"Remote web command access requires authentication; configure JOBBOT_WEB_AUTH_TOKENS or JOBBOT_WEB_AUTH_TOKEN, or pass auth tokens when allowRemoteAccess is enabled."

 ❯ test/web-log-transport.test.js:30:7
     28|         logTransport,
     29|       }),
     30|     ).toThrow(/https/i);
       |       ^
     31|   });
     32| 

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/5]⎯

 FAIL  test/web-server-host-guard.test.js > web server host guard > allows remote hosts when explicitly enabled
Error: Remote web command access requires authentication; configure JOBBOT_WEB_AUTH_TOKENS or JOBBOT_WEB_AUTH_TOKEN, or pass auth tokens when allowRemoteAccess is enabled.
 ❯ startWebServer src/web/server.js:8856:11
    8854|   const normalizedAuth = normalizeAuthOptions(authConfig);
    8855|   if (allowRemote && !isLoopbackHost(host) && !normalizedAuth) {
    8856|     throw new Error(
       |           ^
    8857|       "Remote web command access requires authentication; configure JO…
    8858|         "or JOBBOT_WEB_AUTH_TOKEN, or pass auth tokens when allowRemot…
 ❯ test/web-server-host-guard.test.js:30:40

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[2/5]⎯

 FAIL  test/web-server-host-guard.test.js > web server host guard > accepts truthy environment values for remote access
Error: Remote web command access requires authentication; configure JOBBOT_WEB_AUTH_TOKENS or JOBBOT_WEB_AUTH_TOKEN, or pass auth tokens when allowRemoteAccess is enabled.
 ❯ startWebServer src/web/server.js:8856:11
    8854|   const normalizedAuth = normalizeAuthOptions(authConfig);
    8855|   if (allowRemote && !isLoopbackHost(host) && !normalizedAuth) {
    8856|     throw new Error(
       |           ^
    8857|       "Remote web command access requires authentication; configure JO…
    8858|         "or JOBBOT_WEB_AUTH_TOKEN, or pass auth tokens when allowRemot…
 ❯ test/web-server-host-guard.test.js:42:40

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[3/5]⎯

 FAIL  test/web-server-remote-access.test.js > web server remote access guard > permits non-loopback hosts when explicitly enabled
Error: Remote web command access requires authentication; configure JOBBOT_WEB_AUTH_TOKENS or JOBBOT_WEB_AUTH_TOKEN, or pass auth tokens when allowRemoteAccess is enabled.
 ❯ startWebServer src/web/server.js:8856:11
    8854|   const normalizedAuth = normalizeAuthOptions(authConfig);
    8855|   if (allowRemote && !isLoopbackHost(host) && !normalizedAuth) {
    8856|     throw new Error(
       |           ^
    8857|       "Remote web command access requires authentication; configure JO…
    8858|         "or JOBBOT_WEB_AUTH_TOKEN, or pass auth tokens when allowRemot…
 ❯ test/web-server-remote-access.test.js:55:34

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[4/5]⎯

 FAIL  test/web-server-remote-access.test.js > web server remote access guard > permits non-loopback hosts when the environment explicitly opts in
Error: Remote web command access requires authentication; configure JOBBOT_WEB_AUTH_TOKENS or JOBBOT_WEB_AUTH_TOKEN, or pass auth tokens when allowRemoteAccess is enabled.
 ❯ startWebServer src/web/server.js:8856:11
    8854|   const normalizedAuth = normalizeAuthOptions(authConfig);
    8855|   if (allowRemote && !isLoopbackHost(host) && !normalizedAuth) {
    8856|     throw new Error(
       |           ^
    8857|       "Remote web command access requires authentication; configure JO…
    8858|         "or JOBBOT_WEB_AUTH_TOKEN, or pass auth tokens when allowRemot…
 ❯ test/web-server-remote-access.test.js:96:34

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[5/5]⎯


 Test Files  3 failed | 130 passed (133)
      Tests  5 failed | 934 passed (939)
   Start at  23:19:53
   Duration  117.74s (transform 1.14s, setup 0ms, collect 2.54s, tests 114.72s, environment 0ms, prepare 50ms)


Error: AssertionError: expected [Function] to throw error matching /https/i but got 'Remote web command access requires au…'

- Expected: 
/https/i

+ Received: 
"Remote web command access requires authentication; configure JOBBOT_WEB_AUTH_TOKENS or JOBBOT_WEB_AUTH_TOKEN, or pass auth tokens when allowRemoteAccess is enabled."

 ❯ test/web-log-transport.test.js:30:7



Error: Error: Remote web command access requires authentication; configure JOBBOT_WEB_AUTH_TOKENS or JOBBOT_WEB_AUTH_TOKEN, or pass auth tokens when allowRemoteAccess is enabled.
 ❯ startWebServer src/web/server.js:8856:11
 ❯ test/web-server-host-guard.test.js:30:40



Error: Error: Remote web command access requires authentication; configure JOBBOT_WEB_AUTH_TOKENS or JOBBOT_WEB_AUTH_TOKEN, or pass auth tokens when allowRemoteAccess is enabled.
 ❯ startWebServer src/web/server.js:8856:11
 ❯ test/web-server-host-guard.test.js:42:40



Error: Error: Remote web command access requires authentication; configure JOBBOT_WEB_AUTH_TOKENS or JOBBOT_WEB_AUTH_TOKEN, or pass auth tokens when allowRemoteAccess is enabled.
 ❯ startWebServer src/web/server.js:8856:11
 ❯ test/web-server-remote-access.test.js:55:34



Error: Error: Remote web command access requires authentication; configure JOBBOT_WEB_AUTH_TOKENS or JOBBOT_WEB_AUTH_TOKEN, or pass auth tokens when allowRemoteAccess is enabled.
 ❯ startWebServer src/web/server.js:8856:11
 ❯ test/web-server-remote-access.test.js:96:34


Step "vitest" failed with exit code 1.
Error: Process completed with exit code 1.
```

new codex task, not a r/e/v/i/e/w task

--- a/test/web-log-transport.test.js
+++ b/test/web-log-transport.test.js
@@ -23,6 +23,7 @@ describe('web server log transport', () => {
         host: '0.0.0.0',
         port: 0,
         allowRemoteAccess: true,
+        auth: { tokens: ['log-transport-token'] },
         csrfToken: 'log-transport-csrf',
         commandAdapter: {},
         logTransport,

--- a/test/web-server-host-guard.test.js
+++ b/test/web-server-host-guard.test.js
@@ -30,6 +30,7 @@ describe('web server host guard', () => {
     const server = await startWebServer({
       host: '0.0.0.0',
       port: 0,
+      auth: { tokens: ['host-guard-token'] },
       csrfToken: 'host-guard-csrf',
       commandAdapter: {},
     });
@@ -42,6 +43,7 @@ describe('web server host guard', () => {
     const server = await startWebServer({
       host: '0.0.0.0',
       port: 0,
+      auth: { tokens: ['host-guard-env-token'] },
       csrfToken: 'host-guard-csrf',
       commandAdapter: {},
     });

--- a/test/web-server-remote-access.test.js
+++ b/test/web-server-remote-access.test.js
@@ -56,6 +56,7 @@ describe('web server remote access guard', () => {
       host: '0.0.0.0',
       port: 0,
       allowRemoteAccess: true,
+      auth: { tokens: ['remote-guard-token'] },
       audit: { logPath: path.join(tempDir, 'audit.jsonl') },
       csrfToken: 'remote-guard-csrf',
       commandAdapter: {},
@@ -96,6 +97,7 @@ describe('web server remote access guard', () => {
     server = await startWebServer({
       host: '0.0.0.0',
       port: 0,
+      auth: { tokens: ['remote-env-token'] },
       audit: { logPath: path.join(tempDir, 'audit.jsonl') },
       csrfToken: 'remote-guard-csrf',
       commandAdapter: {},

